README.md

```diff
@@ -1,42 +1,61 @@
-Thank you very much for taking the time to report a bug to us, we greatly
-appreciate it. Even more so, since you are about to take the time to create an
-[MCVE (Minimal Complete Verifiable Example)](https://stackoverflow.com/help/minimal-reproducible-example). Thanks to you, we can make jOOQ an
-even better product!
+Thank you very much for taking the time to report a bug to us, we greatly appreciate it.
+Even more so, since you are about to take the time to create an
+[MCVE (Minimal Complete Verifiable Example)](https://stackoverflow.com/help/minimal-reproducible-example).
+Thanks to you, we can make jOOQ an even better product!
+
+
+This project contains —for each supported language (Java, Kotlin, Scala) and SQL dialect (H2, MySQL, Oracle, PostgreSQL, SQL Server)—
+an minimal project from which you can derive your MCVE. Each of these projects sits in its own folder and:
+
+- Uses testcontainers (for MySQL, Oracle, PostgreSQL, SQL Server) to set up a database instance, and makes it available to jOOQ and JUnit.
+- Installs a sample schema located in `src/main/resources/db/migration` into an file-based H2 database or testcontainers managed MySQL, Oracle, PostgreSQL, SQL Server database.
+- Runs jOOQ's code generator on it.
+- Runs a simple integration test.
+
+This should work without any additional setup on your side, other than having a JDK, build tool (only when using Maven), and Docker available.
+If you need to have the commercial distribution of jOOQ installed, e.g. for using Oracle or SQL Server, that must be installed as well.
+In case you do not have Docker, you can still run the H2 based examples directly, ignoring the MySQL, Oracle, PostgreSQL, SQL Server based ones.
+
 
 ## How to use this project to prepare your MCVE
 
-Create a fork from this project and then
+
+First you need to create a fork of this project (do this from the [GitHub web UI](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo))
+so you can publish your MCVE once done. Then clone the repository you just forked and `cd` into it:
+
```

Comment:
Here, too, I understand why someone might want to do this. But "forking" is a very well known concept on github, and I just don't want to create noise about well-known concepts. We don't have to explain git either, or pull, or `cd`, I think?

Besides, such a link is quickly outdated, and I'll just have to maintain it going forward.